src/Conversion/ONNXToKrnl/Tensor/Range.cpp

```diff
@@ -113,6 +113,7 @@ struct ONNXRangeOpLowering : public ConversionPattern {
     }
 
     // Create a single loop.
+    LiteralIndexExpr zeroIE(0);
```

Comment:
[nit]: I would prefer to reduce the live range for this expression (declare closer to its use).

Reply:
I intentionally define it before `BuildKrnlLoop`, note that krnlLoop will be change to use the builder based interface.

Reply:
ok so is future proof. that is fine.